scripts/import-npa/natural_protected_areas.sh

```
@@ -0,0 +1,461 @@
+  
+  # Parse command line arguments
+  parse_args "$@"
+  
+  # Check required tools
+  check_environment || exit 1
+  
+  # Prepare temporary directory
+  prepare_temp_dir
+  
+  # Download dataset
+  local zip_path=$(download_dataset)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)